app/controllers/users/invitations_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Users::InvitationsController < Devise::InvitationsController
```

Comment:
@amarkpark was talking about refactoring admin stuff under an admin namespace.  You guys might want to sync up to talk about if it makes sense to do this here too.  If we're doing it in one place, we should do this in both places.